prod-aws/account-platform/topics.tf

```diff
@@ -24,3 +23,116 @@ resource "kafka_topic" "account_exceptions_events" {
   partitions         = 15
   replication_factor = 3
 }
+
+import {
+  to = kafka_topic.legacy_account_events_compacted
+  id = "legacy.account.events.compacted"
+}
+
+resource "kafka_topic" "legacy_account_events_compacted" {
+  config = {
+    "cleanup.policy" = "compact"
+    "segment.ms"     = "604800000"
+  }
+  name               = "legacy.account.events.compacted"
+  partitions         = 50
+  replication_factor = 3
+}
+
+import {
+  to = kafka_topic.legacy_account_changelog_events
+  id = "legacy.account.changelog.events"
+}
+resource "kafka_topic" "legacy_account_changelog_events" {
+  config = {
+    "cleanup.policy"        = "compact"
+    "max.compaction.lag.ms" = "604800000"
+  }
+  name               = "legacy.account.changelog.events"
+  partitions         = 15
+  replication_factor = 3
+}
+
+import {
+  to = kafka_topic.finance_events_compacted
+  id = "finance.events.compacted"
+}
+resource "kafka_topic" "finance_events_compacted" {
+  config = {
+    "cleanup.policy"        = "compact"
+    "max.compaction.lag.ms" = "604800000"
+  }
+  name               = "finance.events.compacted"
+  partitions         = 15
+  replication_factor = 3
+}
+
+import {
+  to = kafka_topic.legacy_account_change_events_compacted
+  id = "legacy.account.change.events.compacted"
+}
+resource "kafka_topic" "legacy_account_change_events_compacted" {
+  config = {
+    "cleanup.policy"        = "compact"
+    "max.compaction.lag.ms" = "604800000"
+  }
+  name               = "legacy.account.change.events.compacted"
+  partitions         = 50
```

Comment:
Not necessarily related to the PR but 50 partitions sounds like a lot, just curious how you guys decided on this config? 🙇

Reply:
It´s a good question to keep asking. 
We don´t necessarily need 50 for this one.
But for very large topics with a lot of consumers like legacy.account.events, it helps speed up consumption.

I think (maybe) when compacting with a relay you need to match partitions in source and target, but can´t remember if that´s true.

Before we migrate these topics let´s revisit partition numbers